### ISSUE TITLE
Internet Radios Don't Work

### ISSUE THREAD

**Issue by fennifith** [fennifith/Alarmio#5]
Pretty sure this is just a bad implementation of ExoPlayer. I'll need to spend a couple minutes to do a proper dive into its documentation when I get the time and check that I'm doing everything properly.

**Comment by bestouff** [comment]
Do they now ? I tried with http://direct.franceinter.fr/live/franceinter-midfi.mp3 and http://ledjamradio.ice.infomaniak.ch/ledjamradio.mp3 and none of them worked.

**Comment by raboof** [comment]
@bestouff the current version of Alarmio supports streams supported by the HLS media source. Those streams are likely supported by the Progressive media source. I ran into the same thing and prepared a pull request: #87

**Comment by raboof** [comment]
(perhaps can be closed now that #87 is merged?)

**Comment by bestouff** [comment]
(maybe close it only once it's in a released version)

**Comment by fennifith** [comment]
Yeah, I didn't realize #86 was a duplicate of this. Keep it open until it's merged into `master`, that way it stays visible for people encountering the issue until a new version is released.
